docs/ruleset_functions/is_baseline_system_1(HVAC_id).md

```diff
@@ -0,0 +1,35 @@
+# is_baseline_system_1  
+
+**Description:** Get a TRUE or FALSE output as to whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 1.  
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled as system 1 in the B_RMR.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_baseline_system_1**: The functions returns TRUE if the system is system 1 and FALSE if the system is not system 1.  
+ 
+**Function Call:** 
+1. get_hvac_zone_list_w_area()   
+
+## Logic:  
+- Set the is_baseline_system_1 boolean variable to true (if it does not meet the criteria below then it gets set to false): `is_baseline_system_1 = TRUE`  
+- Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
```

Comment:
do we need a step here to cycle through each HVAC system in the B-RMD?

Reply:
@claperle I was thinking the cycling would be done in the master function which would pass the ids to the sub functions which would return true or false.

Reply:
@claperle makes sense. At a more higher level, are we assuming we would reach the b RMD, find each HVAC system and then loop through the function of all n baseline systems to categorize the system type?

Another way I can think about writing this function is to have a single function which starts with

- Is the system serving SZ or MZ?
- If MZ, is the fan VAV?
- If MZ= True and Fan VAV = True, is the terminal unit RH HW or PIU electric 
- Then we check cooling source and heating source
Etc.
I can see this getting very complicated so I guess I have talked myself out of this approach.

Reply:
@supriyagoel LOL, yeah exactly, I think that is the reason it was suggested to do it this may just to break it into more manageable pieces.

Reply:
Oh, and the answer to your first Q is yes.

Reply:
I don't have strong feelings one way or the other, but the approach Supriya suggested still uses sub-functions (i.e., breaks things into managable pieces) without pursuing known dead ends. E.g., if the system is MZ, we know that these have to be systems 5-8 so won't need to check for all other system types.  And the logic will follow exactly this diagram https://onedrive.live.com/?authkey=%21ACtFTkgcGDgG1Qk&cid=796769604E905084&id=796769604E905084%21387&parId=796769604E905084%21277&o=OneUp except each diamond will be replace with a call to corresponding sub-function.

Reply:
@supriyagoel, to your first question, the proposed function would call subfunctions to determine system type for each of the baseline HVAC systems. It can be called for each baseline HVAC system to create the system library, exactly as chritina has it now. 

To your second question, yes, I suggest updating the diagram to include additional checks and HVAC systems, so that the diagram fully describes the mapping logic.